internal/models/ingrediente.go

```diff
@@ -0,0 +1,26 @@
+package models
+
+import "time"
+
+type Ingrediente struct {
+	id              int
+	nombre          string
```

Comment:
¿Podéis por favor mirar el concepto de objeto valor? ¿Un objeto valor tiene ID?

Reply:
Es verdad que en el DDD los objetos de valor solo son definidos por sus atributos. Bajo este pretexto dos objetos con los mismos atributos son iguales. Por ende Un objeto de valor no debería tener un id ya que impide esta metodología